### PR TITLE
Fixed bash typos!!

### PR DESCRIPTION
**Pull request by ejmoore99**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -147,11 +147,95 @@ Enhancement suggestions are tracked as [GitHub issues](https://github.com/lucava
...
+- Submit the pull request.
 
 ### Improving The Documentation
 
-TBD
+Good documentation is crucial for any project. Here’s how you can help improve it:
+
+**1. Identify Areas for Improvement:**
+
+-Look for typos, unclear sections, missing information, or outdated content.
```
Comment (lucavallin) [inline CONTRIBUTING.md:210]: This does not render correctly.

**Comment by lucavallin** [comment]
Hey @ejmoore99, thanks for the update. I left a comment. Note that this could have been a new commit in the previous PR rather than a whole new PR.

**Comment by ejmoore99** [comment]
> Hey @ejmoore99, thanks for the update. I left a comment. Note that this could have been a new commit in the previous PR rather than a whole new PR.

Thank you for your feed back. I have fixed the text so that it should render correctly.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -147,11 +147,95 @@ Enhancement suggestions are tracked as [GitHub issues](https://github.com/lucava
...
+Good documentation is crucial for any project. Here’s how you can help improve it:
+
+**1. Identify Areas for Improvement:**
+
+- Look for typos, unclear sections, missing information, or outdated content.
+
+**2. Make Edits:**
+
+- Fork and clone the repository as described above.
+- Navigate to the documentation files, often located in a docs folder or as README.md files.
```
Comment (lucavallin) [inline CONTRIBUTING.md:215]: This is the generic stuff ChatGPT produces ;) can you make it specific for the project?

**Comment by lucavallin** [comment]
no updates

**Pull request closed without merging**